posthog/settings/data_stores.py

```diff
@@ -100,10 +100,20 @@ def postgres_config(host: str) -> dict:
 read_host = os.getenv("POSTHOG_POSTGRES_READ_HOST")
 if read_host:
     DATABASES["replica"] = postgres_config(read_host)
+    DATABASE_ROUTERS = ["posthog.dbrouter.ReplicaRouter"]
```

Comment:
a comment somewhere please that this doesn't work for raw connections created using `connection.cursor()` so people would have to think in those cases themselves :)

Reply:
I'm going to write a doc to accompany this change, once it's ready for review/gone in!

Reply:
but will also do a comment

Reply:
yeah sorry if this is nitty / too early for draft, just wanted to mention so it doesn't slip :)

Reply:
Totally cool, appreciate it! 🙏